src/components/Personality.js

```diff
@@ -48,9 +93,10 @@ class Personality extends Component {
               this.setState({ values: newValues });
             }}
 
-            maximumValue={4}
-            minimumValue={0}
+            maximumValue={2}
```

Comment:
Range 0~4 also works, But I think to represent neutral value, 0 is more straightforward than 2 and also easy to determine which one is close among both opposite personality